ding/model/template/maqac.py

```diff
@@ -91,7 +91,7 @@ def __init__(
             )
 
     def forward(self, inputs: Union[torch.Tensor, Dict], mode: str) -> Dict:
-        r"""
+        """
```

Comment:
You need to polish `maqac` with comparsion to `qac`

Reply:
fixed